.rat-excludes

```diff
@@ -166,8 +166,14 @@ VariableLengthPKTest.java
 # asciidoc extention file -- MIT license
 google-analytics-postprocessor.rb
 
-# installer JSON config files
+# pyinstaller JSON config files
 mod_cfgs.json
 prompt.json
 script.json
 version.json
+
+# pyinstaller dependencies
+prettytable.py
+socks.py
+iri2uri.py
+__init__.py
```

Comment:
These files all have headers. RAT is only confused by two of them (prettytable & socks). So the others should NOT be listed in this file.

Also, comment here should not just state what they are, but why they should be excluded from check -- they have BSD-3 license that RAT does not recognize.